src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
There was a reference back to the init file which kind of took us away from the actual issue.  I think the init file legendUrl field is something we should avoid.  How it's done in the arcgis....js file can be whatever is consistent and easy.